users/config.rst

```diff
@@ -792,11 +792,12 @@ setLowPriority
 Listen Addresses
 ^^^^^^^^^^^^^^^^
 
-The following address types are accepted in sync protocol listen addresses:
+The following address types are accepted in sync protocol listen addresses
+(multiple can be specified by comma separation):
```

Comment:
I think its space separated? Or could be either

Reply:
In the code here:
https://github.com/syncthing/syncthing/blob/3cb7b8f22bf254a5270acdce52686863979cf35d/gui/app.js#L52
and here:
https://github.com/syncthing/syncthing/blob/3cb7b8f22bf254a5270acdce52686863979cf35d/gui/app.js#L174
I think (if my understanding is correct) that comma-separated is the only accepted format, at least in the GUI.

Reply:
You are right.